torch/onnx/__init__.py

```diff
@@ -103,11 +103,17 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
         export_params (bool, default True): if True, all parameters will
             be exported. Set this to False if you want to export an untrained model.
             In this case, the exported model will first take all of its parameters
-            as arguments, with the ordering as specified by ``model.state_dict().values()``
+            as arguments, with the ordering as specified by ``model.state_dict().values()``.
+            This helps on stripping parameters from the model so that large models could be export
```

Comment:
Few minor issues in this comment. Consider:
"This helps in stripping parameters from the model so that large models could be export successfully. Besides this, if this is False, no matter the model will be export in inference mode or training mode, the optimization which may adjust graph inputs will be skipped—for example, Conv and BatchNorm fusion."